src/java.base/share/classes/jdk/internal/foreign/SystemLookup.java

```diff
@@ -64,11 +65,13 @@ private SystemLookup() { }
 
     private static SymbolLookup makeSystemLookup() {
         try {
-            return switch (CABI.current()) {
-                case SYS_V, LINUX_AARCH_64, MAC_OS_AARCH_64, LINUX_RISCV_64, SYS_V_PPC_64LE, SYS_V_S390X -> libLookup(libs -> libs.load(jdkLibraryPath("syslookup")));
-                case AIX_PPC_64 -> makeAixLookup();
-                case WIN_64, WIN_AARCH_64 -> makeWindowsLookup(); // out of line to workaround javac crash
-            };
+            if (Utils.IS_AIX) {
+                return makeAixLookup();
+            } else if (Utils.IS_WINDOWS) {
+                return makeWindowsLookup();
```

Comment:
Please add the AIX case after the Windows case to minimize the change relative to upstream:
```java
            if (Utils.IS_WINDOWS) {
                return makeWindowsLookup();
            } else if (Utils.IS_AIX) {
                return makeAixLookup();
```

Reply:
Fixed as suggested above.